en/docs/fragments/pickup-dispatch-oidc.md

```diff
@@ -3,11 +3,11 @@
 
 ### Register a service provider
 
-1. Log in to the [Management Console](insertlink) using admin/admin credentials. 
+1. Log in to the Management Console (https://<IS_HOST\>:<PORT\>/carbon) using admin/admin credentials. 
```

Comment:
instead of this (https://<IS_HOST\>:<PORT\>/carbon) shall we use *`(https://<IS_HOST>:<PORT>/carbon)`*